api/Document.json

```diff
@@ -6963,10 +6983,10 @@
               "version_added": false
             },
             "firefox": {
-              "version_added": true
+              "version_added": "10"
```

Comment:
Some event handler attributes added starting here. The form sensible groups, and the versions are way in the past but still plausible, so I haven't tried to verify further.